cartography/intel/github/__init__.py

```diff
@@ -47,8 +46,3 @@ def start_github_ingestion(neo4j_session, config):
             )
         except exceptions.RequestException as e:
             logger.error("Could not complete request to the GitHub API: {}", e)
-    run_cleanup_job(
```

Comment:
This is the main change: moving this cleanup function out of the `__init__` and splitting it up into `github.users` and `github.repos`.